pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[tool.poetry]
+name = "nornir"
+version = "2.0.0"
+description = "Pluggable multi-threaded framework with inventory management to help operate collections of devices"
+authors = ["David Barroso <dbarrosop@dravetech.com>"]
+readme = "README.md"
+repository = "https://github.com/nornir-automation/nornir"
+classifiers = [
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+]
+
+[tool.poetry.dependencies]
+python = ">= 3.6, < 3.8"
+colorama = "^0.4.1"
+jinja2 = "^2.10"
+napalm = "^2.3"
+netmiko = "^2.3"
+paramiko = "^2.4"
+requests = "^2.21"
+"ruamel.yaml" = "^0.15.85"
+mypy_extensions = "^0.4.1"
+pydantic = "^0.17.0"
```

Comment:
I think the versioning should be evaluated on per package basis..
For example, I'd propose to keep `requests` as `*`. Something less stable as `pydantic` could be pinned more aggressively. The main reasoning why - when a new (major) version of something like `requests` comes, nornir users will not be able to update to it, until we change in `nornir`

Reply:
What's the point of using something like poetry then?

Reply:
Just to double-check, are you ok with versions >= 2.0.0 and <= 3.0.0? I thought you would leave something like ^2.18.1 so that it translates to >= 2.18.1 and <= 3.0.0